src/explore-education-statistics-common/src/components/Link.tsx

```diff
@@ -0,0 +1,34 @@
+import React, {
+  AnchorHTMLAttributes,
+  FunctionComponent,
+  ReactNode,
+} from 'react';
+
+type Props = {
+  className?: string;
+  to: string;
+  children: ReactNode;
+} & AnchorHTMLAttributes<HTMLAnchorElement>;
+
+type RendererType = FunctionComponent<Props>;
+
+// some default renderer
+let Renderer: FunctionComponent<Props> = ({ children }: Props) => {
+  return (
+    <span>
+      <em>Link Renderer Not Defined</em>
+      {children}
+    </span>
+  );
+};
+
+const Link: FunctionComponent<Props> = (props: Props) => {
+  return <Renderer {...props} />;
+};
+
+export function RegisterLinkRenderer<Props>(component: RendererType) {
```

Comment:
Would go with camelcased `registerInternalLink` as its a function. Only components and classes should be uppercased by convention.